membership-attribute-service/app/components/TouchpointComponents.scala

```diff
@@ -0,0 +1,43 @@
+package components
+
+import com.gu.config.{Membership, DigitalPack}
+import com.gu.membership.salesforce.SimpleContactRepository
+import com.gu.membership.stripe.StripeService
+import com.gu.membership.touchpoint.TouchpointBackendConfig
+import com.gu.membership.zuora.{SubscriptionService, rest, soap}
+import com.gu.monitoring.ServiceMetrics
+import com.gu.services.ZuoraPaymentService
+import configuration.Config
+import configuration.Config.SalesforceConfig
+import play.api.libs.concurrent.Akka
+import play.api.libs.ws.WS
+import repositories.MembershipAttributesSerializer
+import play.api.Play.current
+import services.{AttributeService, DynamoAttributeService}
+import play.api.libs.concurrent.Execution.Implicits._
+
+trait TouchpointComponents {
+
+  val stage: String
+  val sfConfig: SalesforceConfig
+
+  lazy val conf = Config.config.getConfig("touchpoint.backend")
```

Comment:
Not sure it is a good idea to make these lazy, as this will make harder to spot configuration errors which would be otherwise evident right away.